Resources/Styles/package.scss

```diff
@@ -35,6 +35,11 @@
           background-repeat: no-repeat;
           background-position: top 2px left;
         }
+        
+        li.history {
+          grid-column-start: span 2;
+          background-image: var(--image-clock);
```

Comment:
So we don't have a source for icons. I just draw them! 😬

I was going to draw a new one for this, but actually I think `var(--image-warning)` works well here.